java/src/main/java/com/google/appengine/tools/pipeline/impl/tasks/FanoutTask.java

```diff
@@ -1,148 +0,0 @@
-// Copyright 2011 Google Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License"); you may not
-// use this file except in compliance with the License. You may obtain a copy of
-// the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-// License for the specific language governing permissions and limitations under
-// the License.
-
-package com.google.appengine.tools.pipeline.impl.tasks;
-
-import static com.google.appengine.tools.pipeline.impl.util.StringUtils.UTF_8;
-
-import com.google.cloud.datastore.Key;
-import com.google.appengine.tools.pipeline.impl.QueueSettings;
-import com.google.appengine.tools.pipeline.impl.util.GUIDGenerator;
-
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-import java.util.Properties;
-
-/**
- * A Task that represent a set of other tasks.
- * <p>
- * The purpose of this class is to get around two limitations in App Engine:
- * <ol>
- * <li>Only a small number (currently 5) of task queue tasks is permitted to be
- * part of of a data store transaction.
- * <li>Named task queue tasks may not be part of a data store transaction.
- * </ol>
- *
- * This task is used as part of a strategy to get around those limitations:
- * Instead of enqueueing a set of tasks, a single {@code FanoutTask} may be
- * enqueued that, when handled, will cause a collection of other tasks to be
- * enqueued.
```

Comment:
since we're no longer transactionally enqueuing tasks, this `FanoutTask` is no longer serving a purpose